app/models/spree/shipping_method.rb

```diff
@@ -34,6 +34,8 @@ class ShippingMethod < ApplicationRecord
 
     after_save :touch_distributors
 
+    after_initialize :init
+
```

Comment:
I would have put this hook above `after_save` to have a chronological order.

Reply:
I never used this callback before. I hope it works as expected. Calling this in the controller would not have been enough? The fewer callbacks the better.

Reply:
Okay, good for consistency.